pex/pep425tags.py

```diff
@@ -19,11 +18,13 @@
 _OSX_ARCH_PAT = re.compile(r'(.+)_(\d+)_(\d+)_(.+)')
 
 
+# TODO(John Sirois): Kill this file and use wheel.pep425tags from our vendored wheel instead.
+# The `# Issue #1074` doesn't even refer to a pex issue :/.
 def get_config_var(var):
   try:
     return sysconfig.get_config_var(var)
   except IOError as e:  # Issue #1074
-    warnings.warn("{0}".format(e), RuntimeWarning)
+    logger.warn(str(e))
```

Comment:
To clarify, you use `logger.warn` here because the issue does not come from Pex but is copied code from another library? So you do not want `--no-emit-warnings` to silence this, right?

Reply:
No. The source code of functions in this module are plucked at runtime to use in interpreter identification. As such, the functions must be stdlib-only. I'll add a comment about this pre-existing wonkiness.